zerver/lib/events.py

```diff
@@ -356,6 +357,9 @@ def fetch_initial_state_data(
     if want("realm_emoji"):
         state["realm_emoji"] = realm.get_emoji()
 
+    if want("emoji_data_url"):
+        state["emoji_data_url"] = staticfiles_storage.url("generated/emoji/emoji_api.json")
```

Comment:
Maybe add a comment referencing EMOJI_API_FILE_PATH and giving an example of what this will look like?

I realize most of the adjacent things here don't have something like that, but this one is somewhat unique in that it doesn't point to another function (which might have an informative name, docstring, or something). 

I think most readers (myself included) don't know off the top of their head what sort of thing `staticfiles_storage.url` returns.

I suppose the other idea I have, which might be better, is to have this call a one-line function from `zerver/lib/emoji.py` that has this information in a docstring.